### DISCUSSION TITLE
Soroban Live Quest 4 - Cross Contract Explanation Video

### DISCUSSION THREAD

**Discussion by rawritude** [stellar/sorobanathon#64] [General]
Hey Stellar peeps!

In the following video I go over the different components of a cross contract.  Building on the Auth Store quest, I explain how to store data on a contract then retrieve it through another.

[Link to video](https://youtu.be/nyGFAA8hcDE)

<img width="877" alt="image" src="https://user-images.githubusercontent.com/7675978/203481141-0dff16e7-a8a7-42d1-966a-10d471e357b5.png">

**Comment by wildework** [comment]
Video walkthroughs are great, thanks for sharing! I'm going to use this to catch up on the Stellar Quests.

**Comment by achallagundla** [comment]
Heya @rawritude  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 3000 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya
